Source/WTF/wtf/text/CString.h

```diff
@@ -64,6 +65,7 @@ class CString final {
     WTF_EXPORT_PRIVATE CString(const char*);
     WTF_EXPORT_PRIVATE CString(std::span<const char>);
     CString(std::span<const uint8_t>);
+    CString(std::span<const char8_t> characters) : CString(byteCast<uint8_t>(characters)) { }
```

Comment:
Kind of irritating that this "forgets" the character encoding, but it's good for now.